src/main/java/com/epam/ta/reportportal/core/analyzer/auto/impl/SearchLogServiceImpl.java

```diff
@@ -108,6 +108,7 @@ public SearchLogServiceImpl(ProjectRepository projectRepository, LaunchRepositor
 
 		searchRq.setFilteredLaunchIds(searchCollectorFactory.getCollector(searchMode).collect(request.getFilterId(), launch));
 
+		//TODO fix query
```

Comment:
describe what is the problem with this query and how to fix it

Reply:
updated